Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -202,6 +205,12 @@ export default class ReportsView extends Vue {
                 text: "My Notes",
             });
         }
+        if (this.config.modules["Laboratory"]) {
```

Comment:
we used 'AllLaboratory" for lab and "Laboratory" for Covid19

Reply:
Changed.